apps/web-giddh/src/app/material-ledger/ledger.component.html

```diff
@@ -920,16 +946,36 @@ <h3 class="ledger-main-head">
                                                                     class="w-100"
                                                                     (click)="
-                                                                        txn.showDropdown = true;
-                                                                        showBankLedgerPopup(txn, item)
+                                                                        showBankLedgerPopup(txn, item);
+                                                                        txn.showAccountDropdown = true
                                                                     "
                                                                 >
                                                                     <span
                                                                         [ngClass]="{
-                                                                            'account-placeholder': true,
-                                                                            'account-placeholder-text':
+                                                                            'account-placeholder yes': true,
+                                                                            'account-placeholder-text ':
+                                                                                txn.selectedAccount &&
+                                                                                txn.selectedAccount.name
+                                                                        }"
+                                                                        >{{
+                                                                            txn.selectedAccount
+                                                                                ? txn.selectedAccount.name
+                                                                                : commonLocaleData?.app_select_accounts
+                                                                        }}</span
+                                                                    >
+                                                                </div>
+                                                                <div
+                                                                    *ngIf="txn.showAccountDropdown"
+                                                                    class="w-100"
+                                                                    (click)="txn.showDropdown = true"
+                                                                >
+                                                                    <span
+                                                                        [ngClass]="{
+                                                                            'account-placeholder no': true,
```

Comment:
Please check class name
```suggestion
                                                                            'account-placeholder-no': true,
```